test/verify/check-pages

```diff
@@ -114,8 +114,15 @@ WantedBy=default.target
 
         b.switch_to_top()
 
-        # The top bar should not be visible on fedora-atomic because no dashboards
-        if "fedora-atomic" in m.image:
+        if m.image != "rhel-7-4":
+            # The top bar should not be visible on fedora-atomic because no dashboards
+            b.wait_present(".flex-body")
+            if "atomic" in m.image:
+                b.wait_present(".flex-body.single-nav")
+            else:
+                b.wait_not_present(".flex-body.single-nav")
+            b.wait_present("#host-nav")
+        elif "fedora-atomic" in m.image:
```

Comment:
This is a dead branch now, as any atomic image already satisfies `m.image != "rhel-7-4"`.